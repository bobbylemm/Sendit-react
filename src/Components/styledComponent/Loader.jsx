import styled from 'styled-components';

const Loader = styled.img`
  position: absolute;
  top: 40%;
  left: 40%;
  z-index: 100;
`;
export default Loader;
