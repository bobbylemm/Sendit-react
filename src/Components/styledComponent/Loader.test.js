import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Loader from './Loader';

describe('testing the button element', () => {
  it('should mount button', () => {
    const tree = renderer.create(<Loader />).toJSON();
    expect(tree).toMatchSnapshot();
    expect(tree).toHaveStyleRule('position', 'absolute');
  });
});
