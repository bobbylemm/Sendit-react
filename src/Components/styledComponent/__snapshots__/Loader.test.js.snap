// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the button element should mount button 1`] = `
.c0 {
  position: absolute;
  top: 40%;
  left: 40%;
  z-index: 100;
}

<img
  className="c0"
/>
`;
